tests/functional/artifacts/test_previous_version_state.py

```diff
@@ -161,46 +163,18 @@
       agg_time_dimension: created_at
 
 metrics:
-  - name: blue_customers_post_2010
```

Comment:
Did not mean to remove these changes - my mistake while generating `run_results` v4 (using `1.6.latest` branch) - will add back